go/server.go

```diff
@@ -10,11 +10,22 @@ import (
 	"github.com/plaid/plaid-go/plaid"
 )
 
+func init() {
+	if PLAID_PRODUCTS == "" {
+		PLAID_PRODUCTS = "transactions"
+	}
+	if PLAID_COUNTRY_CODES == "" {
+		PLAID_COUNTRY_CODES = "US,CA,GB,FR,ES"
+	}
```

Comment:
Might make more sense to default to `"US"` or `"US,CA"`, since I'd expect it to be pretty unusual for a client to want to initialize Link with five country codes.

Reply:
https://github.com/plaid/quickstart/pull/88/commits/36e4f0ed8ace352daf90f250c7b073edf2068fc7